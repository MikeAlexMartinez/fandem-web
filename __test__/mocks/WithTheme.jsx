import React from 'react';
import { ThemeProvider } from '@material-ui/styles';
import muiTheme from '../../styles/mui-theme';

const WithTheme = ({ children }) => (
  <ThemeProvider theme={muiTheme}>
    {children}
  </ThemeProvider>
);

export default WithTheme;
