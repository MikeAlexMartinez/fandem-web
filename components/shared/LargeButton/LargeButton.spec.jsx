import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import WithTheme from '../../../__test__/mocks/WithTheme';import LargeButtonComponent from './LargeButton';

describe('<LargeButton />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(
      <LargeButtonComponent
        text="Test Prop 1"
        type="Test Prop 2"
        variant="Test Prop 3"
        handler={() => {}}
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
