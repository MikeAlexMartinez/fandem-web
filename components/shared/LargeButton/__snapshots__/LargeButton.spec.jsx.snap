// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LargeButton /> renders and matches the snapshot 1`] = `
<LargeButton
  classes={
    Object {
      "largeButton": "LargeButton-largeButton-1",
    }
  }
  handler={[Function]}
  text="Test Prop 1"
  type="Test Prop 2"
  variant="Test Prop 3"
/>
`;
